### PR TITLE
rename pcrphase binary to pcrextend

### PR DESCRIPTION
**Pull request by poettering**
This renames the pcrphase to pcrextend, reflecting the fact that it hasn't been about boot phases for a longer time, and is probably going to be become even more generic sooner or later. Hence let's rename the binary (but not the units wrapping it) now rather than later.

Also adds support to selecting the PCR index to measure into manually, and documents the TPM2 event log we maintain briefly.

### CODE REVIEW

File: man/systemd-measure.xml [hunk 1]
```diff
@@ -191,7 +191,8 @@
         <listitem><para>Controls which boot phases to calculate expected PCR 11 values for. This takes a
         series of colon-separated strings that encode boot "paths" for entering a specific phase of the boot
         process. Each of the specified strings is measured by the
-        <filename>systemd-pcrphase-initrd.service</filename> and
+        <filename>systemd-pcrphase-initrd.service</filename>,
+        <filename>systemd-pcrphase-sysinit.service</filename> and
```
Comment (keszybz) [inline man/systemd-measure.xml]: Oxford comma please.

File: src/pcrextend/pcrextend.c [hunk 2]
```diff
@@ -35,14 +35,14 @@ static int help(int argc, char *argv[], void *userdata) {
...
-        r = terminal_urlify_man("systemd-pcrphase", "8", &link);
+        r = terminal_urlify_man("systemd-pcrextend", "8", &link);
         if (r < 0)
                 return log_oom();
 
         printf("%1$s  [OPTIONS...] WORD\n"
                "%1$s  [OPTIONS...] --file-system=PATH\n"
                "%1$s  [OPTIONS...] --machine-id\n"
-               "\n%5$sMeasure boot phase into TPM2 PCR 11.%6$s\n"
+               "\n%5$sExtend a TPM2 PCR with boot phase, machine ID or file system ID.%6$s\n"
```
Comment (keszybz) [inline src/pcrextend/pcrextend.c]: Oxford comma please.

**Review by keszybz — CHANGES_REQUESTED** [review]

**Comment by keszybz** [comment]
The only question I have is whether we should provide a compat symlink? Maybe some users have created custom services to measure things and would benefit from that? LGTM otherwise.

**Comment by poettering** [comment]
> The only question I have is whether we should provide a compat symlink? Maybe some users have created custom services to measure things and would benefit from that? LGTM otherwise.

I'd try without. It's kinda new. If people complain we can add one in a bugfix release.

**Pull request merged**